autoload/nerdtree/ui_glue.vim

```diff
@@ -495,6 +496,17 @@ function! s:jumpToSibling(node, forward) abort
     call b:NERDTree.ui.centerView()
 endfunction
 
+" FUNCTION: s:jumpToBookmarks() {{{1
+" moves the cursor to the bookmark listing
+function! s:jumpToBookmarks() abort
+    try
```

Comment:
@ds2606 Hello sir, I'm sorry to see your beautiful PR has been waiting for this long.
I'm Currently going back into old PRs to find useful unmerged ones like yours. Is there any chance that you have an interest in continuing to work on this PR?

My only issue with this code is using try/catch instead of checking whether or not the bookmark section is open or not via this function call:
```vim
b:NERDTree.ui.getShowBookmarks()
```